### PR TITLE
Add 'dump' and 'load' in Python module

### LINKED ISSUE #17: Load from and dump to streams
Have `pytomlpp.load` to parse TOML from a text stream, and `pytomlpp.dump` to serialise TOML to a text stream.

```python
import io, pytomlpp
stream = io.StringIO()
pytomlpp.dump({"spam": 42}, stream)
stream.seek(0)
print(pytomlpp.load(stream))
```

This will then match the functionality of the standard-library JSON module. This could be implemented in either Python or PyBind11, simply calling the existing `pytomlpp.loads` and `pytomlpp.dumps` for now. Further iterations I imagine could use C++ streams.

### PR DESCRIPTION
**Pull request by EpicWink**
Add a Python package which provides `dump` and `load`, and exposes `DecodeError`, `dumps` and `loads`.

Closes #17

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -27,5 +27,10 @@ keywords = toml, parser, serilization, deserialization, serdes
 setup_requires =
     pybind11~=2.5
 zip_safe = false
+packages = find:
+package_dir = =src
```
Comment (marzer) [inline setup.cfg:31]: `= =src`

Typo?

Comment (EpicWink) [reply setup.cfg:31]: Nope. To specify the root source directory for a given module or package, you need to provide the value in a map for the `package_dir` argument to `setup`. To specify a root source directory for all packages and modules, use `*`.

INI and TOML don't work well with * as the leading charcuterie in an unquoted string, so in setup.cfg it was decided that the empty string will replace it. Therefore, you can read it as the empty string before the second `=`, followed by "src/"

Comment (marzer) [reply setup.cfg:31]: Ah, I see. Thanks.

**Comment by bobfang1992** [comment]
Great, this is exactly what I was planning to do! Thanks!

File: src/pytomlpp/_io.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+"""Python wrapper for Toml++ IO methods."""
+
+from . import _impl
+
+
+def dump(data, fl):
```
Comment (bobfang1992) [inline src/pytomlpp/_io.py:6]: Maybe later we can add type annotations to these python functions but for now, it is okay.

Comment (EpicWink) [reply src/pytomlpp/_io.py:6]: Type annotations require Python 3.5, up to you

**Pull request merged**